gap/PolygonalComplexes/constructors.gd

```diff
@@ -1005,3 +1024,183 @@ DeclareOperation( "UmbrellaDescriptorOfSurface", [IsSimplicialSurface] );
 #! @Arguments surface 
 DeclareOperation( "SimplicialSurfaceByUmbrellaDescriptor", [IsList] );
 #! @EndGroup
+#!
+#!
+#! @BeginGroup
+#! @Description
+#!  A normed umbrella descriptor is a special umbrella descriptor of a
+#!  surface, where an umbrella descriptor is described in 
+#! (<Ref Sect="Section_Constructors_Umbrellas"/>).
+#!  The normed umbrella descriptor of a given umbrella descriptor <A>umdesc</A>
+#!  for the face <A>face</A> and an optional list of neighbours <A>neigh</A> 
```

Comment:
should be: neighbours <A>neighbours</A>  or change parameter to neigh

Reply:
Thank you, it is fixed.